### ISSUE TITLE
Rephrasing instruction on configuring app in setting development environment

### ISSUE THREAD

**Issue by bolu-tife** [vector-im/element-web#21674]
#### What would you like to do?
I want to rephrase the instruction on configuring the app (making a copy of the config.sample.json and renaming it as config.json) 
#### Why would you like to do it?
This would help new contributors to set up their development environment with ease
#### How would you like to achieve it?
I would rephrase the instruction

**Comment by Johennes** [comment]
Closing as the linked pull request was abandoned and this issue is somewhat vague on its own.
